bundler/src/bundler/chunk/merge.rs

```diff
@@ -633,10 +635,51 @@ struct ImportMetaHandler<'a, 'b> {
     file: &'a FileName,
     hook: &'a Box<dyn 'b + Hook>,
     is_entry: bool,
+    inline_ident: Ident,
+    occurred: bool,
     err: Option<Error>,
 }
 
 impl VisitMut for ImportMetaHandler<'_, '_> {
+    fn visit_mut_module(&mut self, n: &mut Module) {
+        n.visit_mut_children_with(self);
+
+        if self.occurred {
+            match self.hook.get_import_meta_props(
+                n.span,
+                &ModuleRecord {
+                    file_name: self.file.to_owned(),
+                    is_entry: self.is_entry,
+                },
+            ) {
+                Ok(key_value_props) => {
+                    n.body.insert(
```

Comment:
Can you use `prepend_stmt` from `swc_ecm_utils` instead of `.insert(0, stmt)`?
The utility method exists to prevent inserting statements before `'use strict'`;

Reply:
Done.